generators/server/templates/pom.xml.ejs

```diff
@@ -644,6 +644,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-validation</artifactId>
         </dependency>
+        <dependency>
```

Comment:
do we need this for default configuration ? Or is it specific to gateway without database ?

Reply:
it seems useful for all reactive projects